### DISCUSSION TITLE
Organization Pages site with a wrong URL

### DISCUSSION THREAD

**Discussion by dukelsky** [community/community#21939] [Other Feature Feedback, Questions, & Ideas]
<p>I have a repository named as <em><a href="http://orgname.github.io" rel="nofollow noopener">orgname.github.io</a></em> where <em>orgname</em> is the name of my organization. According to <a href="https://help.github.com/articles/user-organization-and-project-pages/" rel="nofollow noopener">GitHub Help</a> it should be published as <em><a href="https://orgname.github.io" rel="nofollow noopener">https://orgname.github.io</a></em> but it is published as</p>
<p><em><a href="https://orgname.github.io/orgname.github.io" rel="nofollow noopener">https://orgname.github.io/orgname.github.io</a></em></p>
<p>What do I do to publish the repository as <em><a href="https://orgname.github.io" rel="nofollow noopener">https://orgname.github.io</a></em> ?</p>

**Comment by mpboom** [comment, marked as answer]
<p>Are you sure you didn’t misspell the name?</p>

**Comment by dukelsky** [comment]
<p>I looked at the name many times and only after reading your reply I looked at it once more and found that I did misspell the name. Thank you for your help!</p>
